nixos/roles/external_net/default.nix

```diff
@@ -84,9 +84,13 @@ in
     flyingcircus.roles.vxlan.gateway = true;
 
     boot.kernel.sysctl = {
-      "net.ipv4.ip_forward" = fclib.mkOverridePlatformModule 1;
-      "net.ipv6.conf.all.forwarding" = fclib.mkOverridePlatformModule 1;
-      "net.ipv6.conf.default.forwarding" = fclib.mkOverridePlatformModule 1;
+      # Use manual override priorities here to avoid conflicts with
```

Comment:
Verified that, somewhere down the type chain, lib.mergeOneOption is indeed at fault. So this is indeed necessary.